deps/rabbitmq_federation/src/rabbit_federation_exchange_link.erl

```diff
@@ -438,12 +438,8 @@ go(S0 = {not_started, {Upstream, UParams, DownXName}}) ->
                           {longstr, N} -> N;
                           _            -> unknown
                       end,
-              {Serial, Bindings} =
-                  rabbit_misc:execute_mnesia_transaction(
-                    fun () ->
-                            {rabbit_exchange:peek_serial(DownXName),
-                             rabbit_binding:list_for_source(DownXName)}
-                    end),
+              {Serial, Bindings} = {rabbit_exchange:peek_serial(DownXName),
+                                    rabbit_binding:list_for_source(DownXName)},
```

Comment:
Is it ok to get rid of the transaction?

Reply:
I think so. The original implementation of `rabbit_binding:list_for_source` already uses `mnesia:async_dirty`

Reply:
Ok, thank you.